content/en/docs/community_roles/user.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Project User
+hide_readingtime: true
+weight: 10
+---
+
+A Project User engages with the project primarily by utilizing its features, following the guidelines set forth in its license. While they may not contribute directly, their usage helps validate the project’s relevance and functionality, providing passive feedback through adoption and interaction with the software.
+
+Feedback may be provided by users through the form of GitHub _Issues_ or _Discussions_, depending on the preference of the project.
+
```

Comment:
I think we need to call out specific LACK OF GitHub permissions here too.

Reply:
```suggestion
```suggestion

This role does not require any permissions changes.

```
```